db/models/articles.models.js

```diff
@@ -0,0 +1,49 @@
+const connection = require("../connection");
+const { createCommentNumber, checkArticleExists } = require("../seeds/utils");
+
+exports.retreiveArticleById = (article_id) => {
+  return connection
+    .query(` SELECT  *  FROM articles WHERE article_id = $1;`, [article_id])
+    .then((result) => {
+      if (result.rows.length === 0) {
+        return Promise.reject({ status: 404, msg: "Not found!" });
+      }
+      return result.rows;
+    });
+};
+
+exports.retreiveArticles = () => {
+  return connection
+    .query(
+      `SELECT articles.article_id, articles.title, articles.topic, articles.author, articles.created_at, articles.votes, articles.article_img_url FROM articles ORDER BY created_at DESC`
+    )
+    .then((result) => {
+      return createCommentNumber(result.rows);
+    });
+};
+
+exports.retreiveCommentByArticleId = (article_id) => {
+  return checkArticleExists(article_id)
+    .then(() => {
+      return connection.query(
+        `SELECT * FROM comments WHERE article_id = $1 ORDER BY created_at DESC`,
+        [article_id]
+      );
+    })
+    .then((result) => {
+      return result.rows;
+    });
+};
+
+exports.addComment = (newComment, article_id) => {
+  const { body, author, votes, created_at } = newComment;
+  const newCommentQuery = `INSERT INTO comments (body, article_id, author, votes, created_at) VALUES ($1, $2, $3, 0, now()) RETURNING *;`;
```

Comment:
you only need to insert into body, article_id,  and author. Votes and created_at are set as default values in your seed file